ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Thank you for submitting an issue!
+
+Please make sure you include the names of the affected language(s) in your issue title, so that we can get the correct maintainers to look at your issue. Here are some examples:
+
+
+    flatc converts a protobuf 'bytes' field to 'string' in fbs schema file (all languages)    
```

Comment:
maybe put a `-` in front of these lines so it is obvious there are 2 examples, not 1.